package.json

```diff
@@ -63,6 +64,7 @@
     "flow:ci": "flow check --max-workers 0",
     "lint": "eslint --config .eslintrc.js packages",
     "lint:ci": "eslint --config .eslintrc.js",
+    "publish:ci": "node utils/pre-publish-check-ci.js && git diff --stat --exit-code HEAD && yarn build && changeset publish",
```

Comment:
Are you planning to add the `pre-publish-check-ci.js` script in a separate PR?

Reply:
Good catch.  I'll add that in.